core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -229,53 +226,50 @@ class LoadBalancerService(
     def invokerHealth = invokerPool.ask(GetStatus)(Timeout(5.seconds)).mapTo[Map[String, InvokerState]]
 
     /** Return a sorted list of available invokers. */
-    private def getAvailableInvokers(): Array[Int] = {
-        val future = invokerHealth.map { map =>
-            map.collect {
-                case (name, Healthy) => name.drop(7).toInt
-            }.toArray.sorted
-        }.recover {
-            case _ => Array.empty[Int]
-        }
-
-        Await.result(future, 5.seconds)
+    private def availableInvokers = invokerHealth.map { map =>
```

Comment:
i don't think we should aggressively remove (return) types even for private methods.